lms/envs/bok_choy.py

```diff
@@ -76,6 +76,16 @@
 CELERY_ALWAYS_EAGER = True
 CELERY_RESULT_BACKEND = 'djcelery.backends.cache:CacheBackend'
 
+BLOCK_STRUCTURES_SETTINGS = dict(
+    # We have CELERY_ALWAYS_EAGER set to True, so there's no asynchronous
+    # code running and the celery routing is unimportant.
+    # It does not make sense to retry.
+    BLOCK_STRUCTURES_TASK_MAX_RETRIES=0,
+    # These delay values are irrelevent because we should never retry.
+    BLOCK_STRUCTURES_COURSE_PUBLISH_TASK_DELAY=0,
```

Comment:
Nit: the reason `PUBLISH_TASK_DELAY` is irrelevant is because the task is run synchronously in bokchoy, not because we don't retry.  The `DEFAULT_RETRY_DELAY`, on the other hand, _is_ irrelevant because we don't retry.